GoogleCombined/README.md

```diff
@@ -0,0 +1,4 @@
+### Google Internship Project
```

Comment:
I suggest a structure as follows:

e.g.

# MapsAndPlacesDemo

## Description

_Insert brief description of the project. You can mention that this was an internship project._

## Requirements

* A [Google Cloud Platform API key](https://developers.google.com/maps/documentation/ios-sdk/start#get-key) with Maps SDK for iOS and Places SDk for iOS enabled (comment to Eric: any other services that need to be enabled by the developer? Geocoding?)

## Installation

1. Run `pod install`
2. Open `MapsAndPlacesDemo.xcworkspace`